peripherals/tx_wrapper/index.js

```diff
@@ -1,7 +1,7 @@
 const { Api, JsonRpc, RpcError } = require("eosjs");
 const { JsSignatureProvider } = require("eosjs/dist/eosjs-jssig"); // development only
-const fetch = require("node-fetch"); // node only; not needed in browsers
-//const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+// const fetch = require("node-fetch"); // node only; not needed in browsers
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
```

Comment:
Where are we with this change? My understanding was this was needed for something.

Reply:
I'm reverting this change.  Looks like this may be succeeding now that everything else is working.  May have been an ancillary effect that has now been solved with the updates/upgrades to get this all working again.

Reply:
Seems like it is still a required change: https://github.com/eosnetworkfoundation/eos-evm-node/actions/runs/5743501573

So I've reverted the "revert".

We may need to discuss this change.

Reply:
Since this is not used in production, doesn't seem to matter what approach is used. This will be replaced soon, via a different PR, in the integration test with the miner which is what is used in production.

Reply:
The file [package.json](https://github.com/eosnetworkfoundation/eos-evm-node/blob/gh-11-integration-test/peripherals/tx_wrapper/package.json) needs to be consistent. Here are the changes I recommend
- add "type": "module" in your package.json
- use node LTS [v18.12.0](https://nodejs.org/en/blog/release/v18.12.0)
- change syntax to "import('node-fetch')"

If you are on an older version of node and can't upgrade, please reach out and I'll provide the flags that will get this to work. 

The "type" : "module" activates ECMA Script 6. When "type": "module" isn't in your package.json is treated as legacy JS. If the intention was to use common JS please let me know and I will adjust the recommendations accordingly.

Reply:
Attempting to address your comments @ericpassmore here: [1a8317b](https://github.com/eosnetworkfoundation/eos-evm-node/pull/17/commits/1a8317b19890097664d4adfc222e21b7feda6df8)

Reply:
Took many more commits than then one I cited above, but I think I've got this updated now to address your suggestions @ericpassmore

Reply:
My comments are addressed. I'm marking resolved. I did find some minor JS issues, which are optional suggestions. I made the changes in a separate branch and I'll open to PR to merge into this branch. Specifically 
- all import use single-quotes even up the usage between single and double quote
- replaces deprecated `substr` with supported `substring`
- changed `var` to `const` or `let`
- some variables with auto initialized, added `let` or `const` for clarity